server.py

```diff
@@ -9,11 +10,9 @@
     socket.listen()
 
     client, address = socket.accept()
+    output_instance = NetworkTerminalOutput(client)
```

Comment:
Can you use the library? so this will be encapsulated in the constructor?

Reply:
yeah i can use it, successfully downloaded it with pip install

Reply:
Awesome. Add it an ask for a re-review.